ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -29,22 +29,27 @@
 #include <wiredtiger.h>
 #include <wiredtiger_ext.h>
 
-#include <aws/core/Aws.h>
 #include "s3_connection.h"
+#include "s3_log_system.h"
+#include <aws/core/Aws.h>
+#include <aws/core/utils/memory/stl/AWSString.h>
+#include <aws/core/utils/logging/AWSLogging.h>
 
 #define UNUSED(x) (void)(x)
 
 /* S3 storage source structure. */
 typedef struct {
     WT_STORAGE_SOURCE storageSource; /* Must come first */
     WT_EXTENSION_API *wtApi;         /* Extension API */
+    uint32_t awsVerboseLevel;
```

Comment:
Isn't this a WiredTiger verbosity level? The mapping to an AWS log level is done inside the `S3LogSystem` class.

Reply:
Yes named it this as it will be used for the AWS SDK log system but I can see how that would be confusing. I can adjust all the names!